romancal/source_catalog/tests/test_source_catalog.py

```diff
@@ -300,8 +300,8 @@ def test_l2_input_model_unchanged(image_model, tmp_path):
         save_results=False,
     )
 
-    assert_allclose(original_data, image_model.data, atol=5.0e-5)
-    assert_allclose(original_err, image_model.err, atol=5.0e-5)
+    assert_equal(original_data, image_model.data)
+    assert_equal(original_err, image_model.err)
```

Comment:
I think it would be safer to use `assert_allclose()` because both arrays are of type `float32`.

Reply:
With this PR the arrays really are identical, so `assert_equal` should always work.